include/aspect/stokes_matrix_free.h

```diff
@@ -152,7 +152,8 @@ namespace aspect
         void fill_cell_data (const dealii::LinearAlgebra::distributed::Vector<number> &viscosity_values,
                              const double pressure_scaling,
                              const Triangulation<dim> &tria,
-                             const DoFHandler<dim> &dof_handler_for_projection);
+                             const DoFHandler<dim> &dof_handler_for_projection,
+                             const bool for_mg);
```

Comment:
The meaning of this parameter is not clear from its name. What does `for_mg` mean? The whole operator is used in a multigrid cycle. Please rename or explain in the documentation.

Reply:
This variable was needed since for both the `mass_matrix` and `velocity_matrix` needed both active and level versions. We used this variable to construct the correct cell_iterator when filling the viscosity data. 

There is a function in the deal.II `MatrixFree` class that tells us this information. I've removed the input parameter `for_mg` and call to the deal.II function inside the implementation of `fill_cell_data()`.

Reply:
It turns out that the function in the deal.II `MatrixFree` class that I mention is only in the development version of the code (which is why the last tester failed). I've readded the `for_mg` parameter under the name `is_mg_level_data`, and added a short explanation in the documentation.